pandas/core/internals/blocks.py

```diff
@@ -2179,15 +2179,33 @@ def _na_value(self):
     def fill_value(self):
         return tslibs.iNaT
 
+    def to_dense(self):
+        # TODO(DatetimeBlock): remove
+        return np.asarray(self.values)
+
     def get_values(self, dtype=None):
         """
         return object dtype as boxed values, such as Timestamps/Timedelta
         """
         if is_object_dtype(dtype):
-            return lib.map_infer(self.values.ravel(),
-                                 self._box_func).reshape(self.values.shape)
+            values = self.values
```

Comment:
why can't you always call ``.ravel()``?

Reply:
For starters because DTA/TDA doesn't have ravel() (though I think it should (ditto transpose) that just returns self)

Reply:
yeah I agree, can you make an issue about this